pkg/tsdb/azuremonitor/azuremonitor_test.go

```diff
@@ -65,7 +65,16 @@ func TestNewInstanceSettings(t *testing.T) {
 					AzureCloud:   "AzureCustomizedCloud",
 					ClientSecret: "secret",
 				},
-				Settings: types.AzureMonitorSettings{},
+				Settings: types.AzureMonitorSettings{
+					CustomizedCloudSettings: types.AzureMonitorCustomizedCloudSettings{
+						CustomizedRoutes: map[string]types.AzRoute{
```

Comment:
since the setting is already "CustomizedCloudSettings", I think it's redundant to call this "Customized*" again, I would just call this "Routes" and "AzurePortalURL" below